
import sys

class CatchError:
    def __init__(self,message):
        raise Exception(message)
